components/service/firefox-accounts/src/test/java/mozilla/components/service/fxa/FxaDeviceConstellationTest.kt

```diff
@@ -187,8 +190,8 @@ class FxaDeviceConstellationTest {
         val testDevice1 = testDevice("test1", false)
         val testTab1 = TabHistoryEntry("Hello", "http://world.com/1")
         `when`(account.handlePushMessage("raw events payload")).thenReturn(arrayOf(
-            ASAccountEvent.IncomingDeviceCommand(
-                command = IncomingDeviceCommand.TabReceived(testDevice1, arrayOf(testTab1))
+            ASAccountEvent.CommandReceived(
+                command = IncomingDeviceCommand.TabReceived(testDevice1, SendTabPayload(listOf(testTab1), "flowid", "streamid"))
```

Comment:
The bindings generator uses lists, the a-c code here seems to prefer arrays. I'm not familiar enough with the differences in Kotlin to know under which circumstances the different types should be preferred. Should we invest in making the generated bindings accept an array rather than a list here?

Reply:
A list should be fine.